pkg/kubectl/cmd/config/get_clusters.go

```diff
@@ -42,20 +42,27 @@ func NewCmdConfigGetClusters(out io.Writer, configAccess clientcmd.ConfigAccess)
 		Long:    "Display clusters defined in the kubeconfig.",
 		Example: getClustersExample,
 		Run: func(cmd *cobra.Command, args []string) {
-			cmdutil.CheckErr(runGetClusters(out, configAccess))
+			noHeaders := cmdutil.GetFlagBool(cmd, "no-headers")
+			err := runGetClusters(out, configAccess, noHeaders)
+			cmdutil.CheckErr(err)
 		},
 	}
 
+	cmd.Flags().Bool("no-headers", false, "Don't print headers (default print headers).")
```

Comment:
Cobra pflag will print the default value instead.

And the headers here are quite misleading.  I thought that were http headers.

Reply:
Use consistent description as `kubecl get` has for `--no-headers` flag.